development/git_lfs.rst

```diff
@@ -0,0 +1,156 @@
+########################################################
+Using Git Large File Storage (LFS) for Data Repositories
+########################################################
+
+DM uses Git LFS to manage test datasets within our :doc:`normal Git workflow </development/workflow>`.
```

Comment:
The :doc: formatting looks off. It's boxed and uses a monospace font. But it seems like it should be creating a link or something different. Generally monospace style is reserved for referencing command line, code, or other technical specifics.